MOxUnit/@MOxUnitFunctionHandleTestCase/run.m

```diff
@@ -16,6 +16,7 @@
 %
 % NNO 2015
 
+    start_tic = tic;
```

Comment:
No, it only temporarily stores the start time of the current test in the local variable `start_tic`.
MATLAB just handles `tic` without any output arguments in a special way.

The [help for etime](https://uk.mathworks.com/help/matlab/ref/etime.html) explictly says to use tic and toc to time functions and not etime because the system clock is not reliable.